site/content/docs/main/style-guide.md

```diff
@@ -312,6 +328,16 @@ Website navigation links can also be marked up as list items; after all they are
 
 ### Tables
 
-The semantic purpose of a data table is to present tabular data. Sighted users can quickly scan the table but a screen reader goes through line by line. A table caption is used to create a descriptive title for a data table. Assistive technologies (AT) use the HTML table caption element to identify the table contents to the user within the page structure. For example, `<table caption="Do and Don't - Use present tense">`. To make tables accessible, use HTML formatting to create tables.
+The semantic purpose of a data table is to present tabular data. Sighted users can quickly scan the table but a screen reader goes through line by line. A table [caption](https://www.w3schools.com/tags/tag_caption.asp) is used to create a descriptive title for a data table. Assistive technologies (AT) use the HTML table caption element to identify the table contents to the user within the page structure.
+
+If you need to create a table, create the table in markdown and use the table [Hugo shortcode](https://gohugo.io/content-management/shortcodes/) to include a caption.
```

Comment:
👍  thanks for writing this. I'd also note here that we can't use reference links within the shortcodes.